src/main/java/io/fabric8/elasticsearch/plugin/kibana/DocumentBuilder.java

```diff
@@ -54,14 +57,20 @@ public DocumentBuilder intervalName(String name) {
 		return addField("intervalName", name);
 	}
 	
-	public DocumentBuilder applicationFields() {
-		return addField("fields", StringEscapeUtils.escapeJava(APPLICATION_FIELDS));
+	public DocumentBuilder applicationFields(boolean use_cdm) {
+		if ( false == use_cdm )
```

Comment:
This is an odd coding convention for java though I like the explicitness of it.  Consider using !use_cdm but its not a deal breaker for me.